src/components/Trade/types.ts

```diff
@@ -26,7 +26,10 @@ export type TradeAsset = {
   fiatAmount?: string
 }
 
-export type DisplayFeeData<C extends ChainId> = QuoteFeeData<C> & { tradeFeeSource: string }
+export type DisplayFeeData<C extends ChainId> = Omit<QuoteFeeData<C>, 'networkFee'> & {
+  tradeFeeSource: string
+  networkFeeCryptoHuman: string
```

Comment:
eventually we prob want this field in lib but this gets us outta trouble

Reply:
Agree 👌